sealed_sector_health/health.go

```diff
@@ -0,0 +1,25 @@
+package sealed_sector_health
```

Comment:
I don't want to sound too nitpicky, but a shorter name like `sectorhealth` would sound better IMO

Reply:
Understood. I use the full name, `sealed_sector_health` to ensure that the intent is clear: These health-values apply only to sealed (and not staged) sectors.